examples/platform-switching/web-assembly-platform/README.md

```diff
@@ -3,20 +3,25 @@
 To run this website, first compile either of these identical apps:
 
 ```bash
-# Option A: Compile examples/platform-switching/rocLovesWebAssembly.roc
-cargo run -- build --target=wasm32 examples/platform-switching/rocLovesWebAssembly.roc
-
-# Option B: Compile examples/platform-switching/main.roc with `pf: "web-assembly-platform/main.roc"` and move the result
-cargo run -- build --target=wasm32 examples/platform-switching/main.roc
-(cd examples/platform-switching && mv rocLovesPlatforms.wasm web-assembly-platform/rocLovesWebAssembly.wasm)
+# Build roc compiler if you have not done so already
+cargo build
+# Compile the app that uses the Wasm platform
+target/debug/roc build --target=wasm32 examples/platform-switching/rocLovesWebAssembly.roc
```

Comment:
The platform-switching examples are also packaged in the nightly, is it alright if I provide instructions for both the nightly and building from source?

Reply:
Yeah of course!
I've never actually done that, so it's better if you add it! 👍